charts/greenhouse/templates/plugin-thanos.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: greenhouse.sap/v1alpha1
+kind: Plugin
+metadata:
+  name: thanos
+  namespace: {{ .Chart.Name }}
+spec:
+  disabled: false
+  optionValues:
+    - name: testFramework.image.registry
+      value: keppel.global.cloud.sap/ccloud-ghcr-io-mirror
+    - name: thanos.image.repository
+      value: keppel.global.cloud.sap/ccloud-quay-mirror/thanos/thanos
+    - name: thanos.initChownData.image.registry
+      value: keppel.global.cloud.sap/ccloud-dockerhub-mirror
```

We should not use the keppel mirror here directly. Reused the templating from our internally used chart:

```suggestion
    {{- if and .Values.global.quayIoMirror .Values.global.ghcrIoMirror .Values.global.dockerHubMirror }}
    optionValues:
    - name: thanos.image.repository
      value: {{ printf "%s/thanos/thanos" .Values.global.quayIoMirror }}
    - name: thanos.initChownData.image.registry
      value: {{ .Values.global.dockerHubMirror }}
    - name: testFramework.image.registry
      value: {{ .Values.global.ghcrIoMirror }}
    {{- end }}
```